X_mix-tasks.md

```diff
@@ -353,6 +423,51 @@ Interactive Elixir (1.0.4) - press Ctrl+C to exit (type h() ENTER for help)
 iex(1)>
 ```
 
+#### `mix phoenix.digest`
+
+This task does two things - digests our static assets and then compresses them.
```

Comment:
I think it would be a good idea to explain a bit more about what a digest is here. I also think that using "digests" as a verb isn't quite what we want. What do you think about this?

"This task does two things, it creates a digest for our static assets and then compresses them."